activerecord/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-*   Fix encryption of column default values.
+*   Fix a bug where using groups and counts with long table names would return incorrect results.
+
+    *Shota Toguchi*
```

Comment:
I know @Dooor and @yono are a member of Today's "How to Rails Contributing" meeting.
So how about adding co-authored-by for all you guys.

Reply:
Thanks for comment. I added @yono to co-authored-by in commit message and CHANGELOG.md.